pkg/config/services.go

```diff
@@ -334,6 +334,10 @@ var SupportedServices = serviceConfigs{
 			regexp.MustCompile("(?P<FleetRequestId>.*)"),
 		},
 	},
+	{
+		Namespace: "AWS/EC2CapacityReservations",
+		Alias:     "ec2CapacityReservations",
```

Comment:
```suggestion
		Alias:     "ec2CapacityReservations",
		DimensionRegexps: []*regexp.Regexp{
				regexp.MustCompile(":capacity-reservation/(?P<CapacityReservationId>)$"),
		},
```

Reply:
I haven't tested this... do need to define a regexp here in order to capture tagged resources?

Reply:
According to https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonec2.html that's would match the format for a capacity-reservation resource
![image](https://github.com/nerdswords/yet-another-cloudwatch-exporter/assets/4571540/b162dbf8-406b-43d8-a7cb-ffb78d0e1048)

Reply:
Yep, I initially drafted it following that spec. I'll go ahead and merge the PR with this change included, we can always fix later if necessary.